test/jdk/javax/xml/stream/XmlInputFactoryTest.java

```diff
@@ -0,0 +1,37 @@
+package javax.xml.stream;
+
+import org.testng.annotations.Test;
+
+import javax.xml.stream.events.StartDocument;
+
+import java.io.StringReader;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class XmlInputFactoryTest {
```

Comment:
This test is written correctly for regular TestNG, but I don't know if it's "right" for jtreg. I'm happy to make any changes necessary.